recipes/icu/all/conanfile.py

```diff
@@ -1,19 +1,24 @@
 from conan import ConanFile
 from conan.tools.apple import is_apple_os
-from conan.tools.build import cross_building
+from conan.tools.build import cross_building, stdcpp_library
 from conan.tools.env import Environment, VirtualBuildEnv
 from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, mkdir, rename, replace_in_file, rm, rmdir, save
 from conan.tools.gnu import Autotools, AutotoolsToolchain
 from conan.tools.layout import basic_layout
 from conan.tools.microsoft import is_msvc, unix_path
-from conan.tools.scm import Version
-from conans.tools import get_gnu_triplet, sha256sum, stdcpp_library
 import glob
 import os
 import shutil
+import hashlib
 
-required_conan_version = ">=1.53.0"
+required_conan_version = ">=1.57.0"
 
+def sha256sum(file_path):
```

Comment:
```suggestion
def _sha256sum(file_path):
```

Let's keep it as protected, so we can ask to port the original to new imports.